localization/localizationHelpers.js

```diff
@@ -55,6 +55,10 @@ set the value attribute for all elements with a [data-value] attribute
  */
 
 if (typeof document !== 'undefined') {
+  if (languages[getCurrentLanguage()] && languages[getCurrentLanguage()].rtl) {
```

Comment:
Just a small detail but why not use `userLanguage` defined at line 32 instead of calling `getCurrentLanguage()` twice?
```suggestion
  if (languages[userLanguage] && languages[userLanguage].rtl) {
```

Reply:
`userLanguage` is null initially, and is only set when l() is called for the first time. The reason for that, IIRC, is that in the main process, getCurrentLanguage uses app.getLocale, which only works after Electron’s `ready` event. But this code here only runs in the renderer process, so getCurrentLanguage will use the `navigator.language` method, which works immediately.